tests/models/test_tile.py

```diff
@@ -0,0 +1,155 @@
+    mock_open.assert_called_once_with(
+        os.path.join(os.path.dirname(module_file), "../data/apps.yml-etag"), "r"
+    )
+
+
+def test_etag_file_missing(mocker, cdn_transfer):
+    mock_open = mocker.patch("builtins.open", mocker.mock_open())
+    mock_open.side_effect = FileNotFoundError
+
+    assert cdn_transfer._etag() == "12345678"
+
+
+# NOTE: Temporarily disabling this test until we have a reliable means of patching out
+#       `urllib3` calls. Even after multiple attempts, the call to `request` still
+#       tries to resolve the mock domain in the mocked CDN object.
+#       References:
+#       - https://mozilla-hub.atlassian.net/jira/software/c/projects/IAM/issues/IAM-1403
+#       - https://paste.mozilla.org/PsVK4hRz
+#
+# def test_download_config(mocker, cdn_transfer):
+#     mock_response = mock.Mock()
+#     mock_response.status = 200
+#     mock_response.headers = {"ETag": "new-etag"}
+#     mock_response.data = b"mock apps.yml content"
+#     with mock.patch('urllib3.PoolManager') as mock_pool_manager, \
+#          mock.patch('os.fsync') as mock_fsync, \
+#          mock.patch('builtins.open', mock.mock_open()) as mock_open_yml, \
+#          mock.patch('builtins.open', mock.mock_open()) as mock_open_etag:
+#         mock_http = mock_pool_manager.return_value
+#         mock_http.request.return_value = mock_response
+#         mock_open_yml.return_value.fileno.return_value = 3
+#         cdn_transfer._download_config()
+
+#         mock_open_yml.assert_any_call(
+#             os.path.join(os.path.dirname(module_file), "../data/apps.yml"), "wb"
+#         )
+#         mock_open_yml().write.assert_called_once_with(b"mock apps.yml content")
+#         mock_fsync().assert_called_once_with(3)
+#         mock_open_etag.assert_any_call(
+#             os.path.join(os.path.dirname(module_file), "../data/apps.yml-etag"), "w+"
+#         )
+#         mock_open_etag().write.assert_called_once_with("new-etag")
+
+def test_download_config_http_error(mocker, cdn_transfer):
+    mocker.patch("urllib3.PoolManager.request", side_effect=urllib3.exceptions.HTTPError)
+
+    with pytest.raises(urllib3.exceptions.HTTPError):
+        cdn_transfer._download_config()
+
+
+def test_load_apps_yml(mocker, cdn_transfer):
+    mock_open = mocker.patch("builtins.open", mocker.mock_open(read_data="mock apps.yml content"))
+
+    cdn_transfer._load_apps_yml()
+
+    mock_open.assert_called_once_with(
+        os.path.join(os.path.dirname(module_file), "../data/apps.yml"), "r"
+    )
+    assert cdn_transfer.apps_yml == "mock apps.yml content"
+
+
+# NOTE: Temporarily disabling this test until we have a reliable means of patching out
+#       `urllib3` calls. Even after multiple attempts, the call to `request` still
+#       tries to resolve the mock domain in the mocked CDN object.
+#       References:
+#       - https://mozilla-hub.atlassian.net/jira/software/c/projects/IAM/issues/IAM-1403
+#       - https://paste.mozilla.org/PsVK4hRz
```

Comment:
Quibbles but not rejections.
* pastemo links have a lifespan of 'days', this probably needs better recordkeeping if you expect it to live on as a reference.  This paste was "hey I got it to work (maybe? I think?) in another framework"
* something like `@pytest.mark.skip(reason="busted")` rather than a full comment-out might make the recovery/completion diff easier to read in the future, but, that's just me.